src/content/docs/Immediate.Handlers/attributes-and-interfaces.md

```
@@ -0,0 +1,199 @@
+
+| Type                                     | Kind           | Applies to                    |
+| ---------------------------------------- | -------------- | ----------------------------- |
+| `HandlerAttribute`                       | Attribute      | class                         |
+| `BehaviorsAttribute`                     | Attribute      | class, assembly               |
+| `ImmediateAssemblyIdentifierAttribute`   | Attribute      | assembly                      |
+| `IHandler<TRequest, TResponse>`          | Interface      | implemented by generated code |
+| `IStreamingHandler<TRequest, TResponse>` | Interface      | implemented by generated code |
+| `Behavior<TRequest, TResponse>`          | Abstract class | you derive from it            |
+| `StreamingBehavior<TRequest, TResponse>` | Abstract class | you derive from it            |
+
+### `[Handler]`
```

markdownlint-cli2 (0.23.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)